### DISCUSSION TITLE
Sector ratings

### DISCUSSION THREAD

**Discussion by igithepop** [ho-dev/HattrickOrganizer#1666] [Ideas]
I'm wondering for a while now, why don't you just give us back the calculation? Feedback server is still a no-no, and formulas only influemce the numbers for the players in lineup selection. But the predicted sector rating stay the same whatever you do.
One could tuneup the predictor i think before V2, when the versions were sill changing at the third decimal, and ever since it's just missing everything by a mile.
It seems like connecting two dots, but they're constantly not connected.
Thanks

**Comment by wsbrenk** [comment]
there are no secrets in the HO rating calculation. However, I have not yet familiarized myself with the algorithm: 

https://github.com/akasolace/HO/blob/master/src/main/java/core/rating/RatingPredictionManager.java

Your ideas concerning improving this are welcome.
